### ISSUE TITLE
Provide endpoint for `getSentenceVector` to take advantage of pre-trained vectors

### ISSUE THREAD

**Issue by adfi** [messense/fasttext-serving#8]
The `predict` endpoint requires a supervised model and cannot take advantage of [pre-trained vectors](https://fasttext.cc/docs/en/pretrained-vectors.html).   

Please provide an endpoint for `getSentenceVector` to enable use of pre-trained word vectors. Also see discussion in #7 .

**Comment by messense** [comment]
What's the desired input & output for this endpoint?

**Comment by adfi** [comment]
Input: a list of sentences (similar to the example from the README)
Output: a list of N-dimensional vectors (N = 300, most of the pre-trained vectors are 300 dimensional)

**Comment by messense** [comment]
See #9

**Comment by messense** [comment]
Example:

```bash
$ curl -X POST \
     --data "[\"Which baking dish is best to bake a banana bread?\", \"\"]" \
     'http://localhost:8000/sentence-vector' -H 'Content-Type: application/json'
[[0.23871057,-0.16349015,0.22861592,-0.20431939,0.033354294,-0.05835807,-0.15858917,-0.04478525,0.004964519,0.0064614355,0.17566209,0.2218685,-0.119509354,0.019747203,-0.15919454,0.045165718,0.023982493,-0.017125417,-0.24485293,0.0018408481,0.022031585,0.16441615,0.047660455,0.3118913,-0.042062014,-0.0866327,0.04672289,-0.13764994,-0.1909922,0.16869278,-0.35614738,0.035498295,0.1159014,0.07999761,-0.038490385,-0.15970582,-0.31836623,0.22743444,0.047677856,0.2870169,-0.1579064,0.010381979,0.13963734,-0.056227256,0.3776745,-0.19270673,0.093204826,0.14977989,-0.02547033,0.030962486,-0.19536142,0.056752216,-0.042208828,0.059241652,-0.2977422,-0.080795914,0.14286382,-0.096778385,0.061604656,-0.11909055,0.21074258,0.110340916,0.018354924,0.0616959,-0.10974826,0.15869617,-0.0038144428,0.19432752,0.11075088,-0.020449588,-0.021683713,0.20815705,0.02573873,-0.26881516,-0.05332867,-0.06198075,-0.016871734,-0.123823784,0.33198878,-0.122575246,0.1376176,0.17979775,-0.030649582,-0.060948867,-0.09465398,-0.004842651,-0.16939259,0.19171818,0.2573633,0.06491146,0.07803221,0.04874748,0.45925894,-0.17051135,-0.03071333,-0.013879306,-0.2754677,0.070630535,0.17478721,0.010269374],[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]]
```

**Comment by messense** [comment]
Released in 0.6.0-alpha.3.

```bash
cargo install --version 0.6.0-alpha.3 fasttext-serving
```

**Comment by adfi** [comment]
Thanks, it works! Thanks for implementing that so fast :-)
